packages/world/src/utils/PlayerUtils.sol

```diff
@@ -108,7 +81,7 @@ library PlayerUtils {
     if (ReverseMovablePosition._get(coord) != player) {
       return;
     }
-    (EntityId to,) = getOrCreateEntityAt(coord);
+    (EntityId to,) = EntityUtils.getBlockAt(coord);
```

Comment:
Why dont we create the block here like before? ie use `getOrCreateBlockAt`

Reply:
because the entity should exist if the player is already there

Reply:
does it? I though when we move players, we're not actually creating the entity. I think we're only setting the player position, and reading object type, but not actually creating the entity

Reply:
oh wait you are right, will add a comment